intel/intel_chipset.c

```diff
@@ -35,6 +35,7 @@ static const struct pci_device {
 	uint16_t gen;
 } pciids[] = {
 	/* Keep ids sorted by gen; latest gen first */
+	INTEL_RPLS_IDS(12),
```

Comment:
RPL is 13th generation. Could you please check and correct the generation information mapping for all platforms?